storage/buckets/buckets_test.go

```diff
@@ -31,42 +32,77 @@ import (
 	iampb "google.golang.org/genproto/googleapis/iam/v1"
 )
 
+const (
+	testPrefix      = "storage-buckets-test"
+	bucketExpiryAge = time.Hour * 24
+)
+
+func TestMain(m *testing.M) {
+	exit := m.Run()
+
+	tc, _ := testutil.ContextMain(m)
+	// Delete old buckets whose name begins with our test prefix
+	if err := testutil.DeleteExpiredBuckets(tc.ProjectID, testPrefix, bucketExpiryAge); err != nil {
+		// Don't fail the test if cleanup fails
+		log.Printf("Post-test cleanup failed: %v", err)
+	}
+	os.Exit(exit)
+}
+
 func TestCreate(t *testing.T) {
 	tc := testutil.SystemTest(t)
-	bucketName := tc.ProjectID + "-storage-buckets-tests"
+	bucketName := testutil.UniqueBucketName(testPrefix)
+	ctx := context.Background()
+
+	client, err := storage.NewClient(ctx)
```

Comment:
Could we just make this client a global variable for this file and initialize it in `TestMain`?

Reply:
yes, great point!!